platform/core/PolyPodCoreAndroid/core/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'org.jlleitschuh.gradle.ktlint' version '10.2.0'
+}
+
+ktlint {
+    debug = true
+    verbose = true
+    reporters {
+        reporter "checkstyle"
+        reporter "plain"
+    }
+    filter {
+        exclude("**/FlatbuffersGenerated/**")
+    }
+}
+
+android {
+    compileSdk 30
+    buildToolsVersion "30.0.3"
```

Comment:
This [SO answer](https://stackoverflow.com/a/51960054/891440) seems to recommend some `ext` extra properties declared at the top level; that would be `platform`, I guess; that would probably imply carrying over the gradle scripts there; it, however, makes sense to have this kind of thing, since with this we will need integrated build scripts at the platform level. Possibly outside the scope of what is already a huge PR.